web/package.json

```diff
@@ -60,7 +60,7 @@
     "start": "CI=true react-scripts start",
     "start:cypress": "export $(cat .env.cypress | xargs) && react-scripts -r @cypress/instrument-cra start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!ol)/\"",
```

Comment:
Why is this necessary?

Reply:
In WeatherMap we import styling components from OpenLayers, and Jest breaks unless we override `ol` node_modules because of problems with how/when the package is compiled